sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/CryptographicKeyShouldNotBeTooShort.cs

```diff
@@ -34,42 +35,243 @@ namespace SonarAnalyzer.Rules.CSharp
     public sealed class CryptographicKeyShouldNotBeTooShort : SonarDiagnosticAnalyzer
     {
         internal const string DiagnosticId = "S4426";
-        private const string MessageFormat = "Use a key length of at least '{0}' bits.";
+        private const string MessageFormat = "Use a key length of at least {0} bits for {1} cipher algorithm.{2}";
+        private const string AdditionalInfo = " This assignment does not actually update the underlying key size.";
```

Comment:
This constant name seems to generic and it contains clear (single purpose) message. I suggest to rename it to something like `UselessAssignmentInfo`